src/foam/core/Enum.js

```diff
@@ -315,6 +315,12 @@ foam.CLASS({
         return of && of.VALUES[0];
       },
     },
+    {
+      name: 'javaValue',
+      expression: function(of) {
+        return this.of.id + '.' + this.value;
```

Comment:
You don't need to do 'this.of' when 'of' is already a parameter. Also, you should add 'value' as a parameter, so:
expression: function(of, value) { return of.id + '.' value; }